src/index.ts

```diff
@@ -12,6 +12,11 @@ export type EntryStructure = {
   content: string | {} | FolderStructure
 }
 
+export async function createFolderStructureWith(content: string | {} | FolderStructure): Promise<string> {
+  const originalResult = await createFolderStructure({ content })
+  return originalResult.entryPath
```

Comment:
you need to return the cleanup function as well

Reply:
There are libraries that don't return a cleanup function, they rely on the OS cleaning up the temp folder.
So I'd say it's not a requirement, but up to the library author to decide.

Reply:
1. i didn't know the OS cleanups the temp folder. is it occurs in linux and mac? 
2. removing a non-empty folder is non-trivial thing to do if you never did that before (fs.: https://stackoverflow.com/questions/18052762/remove-directory-which-is-not-empty/32197381 , so I provide a working solution.

Update:

 as of node 12, there is a way: `fs.rmdir(dir, { recursive: true })`. so we can skip this. 

Resolved.